### ISSUE TITLE
nest.helpdesk() error: RangeCheck in getinterval_s

### ISSUE THREAD

**Issue by maharjun** [nest/nest-simulator#517]
I have compiled nest without MPI support for python3. Upon running the following

```
import nest
nest.helpdesk()
```

I get the following error

```
In [2]: nest.helpdesk()
---------------------------------------------------------------------------
NESTError                                 Traceback (most recent call last)
<ipython-input-2-846d2128da86> in <module>()
----> 1 nest.helpdesk()

/home/arjun/nest-bleeding-python3/lib/python3.4/site-packages/nest/lib/hl_api_helper.py in stack_checker_func(*args, **kwargs)
    228     def stack_checker_func(*args, **kwargs):
    229         if not get_debug():
--> 230             return f(*args, **kwargs)
    231         else:
    232             sr('count')

/home/arjun/nest-bleeding-python3/lib/python3.4/site-packages/nest/lib/hl_api_info.py in helpdesk(browser)
     68 
     69     sr("/helpdesk << /command (%s) >> SetOptions" % browser)
---> 70     sr("helpdesk")
     71 
     72 

/home/arjun/nest-bleeding-python3/lib/python3.4/site-packages/nest/__init__.py in catching_sli_run(cmd)
     89         engine.run('clear')
     90         raise _kernel.NESTError("{0} in {1}{2}".format(
---> 91             errorname, commandname, message))
     92 
     93 sli_run = hl_api.sr = catching_sli_run

NESTError: RangeCheck in getinterval_s

```

**Comment by heplesser** [comment]
I can reproduce this error under Linux with Python 2.7 with a standard `.nestrc` file containing
```
/helpdesk << /command (firefox) >> SetOptions
```
on a computer that does not have `firefox` installed.

As a work-around, you should change `firefox` to the name of a browser available on your system in the `.nestrc` file in your home directory. The expression in parentheses should start the browser from the command line.

Under OSX 10.11 with Python 3.5, I receive a sensible error message.

When calling `helpdesk` from SLI, the same error occurs with the following material on the stack:
```
SLI ] helpdesk

Oct 12 23:54:46 getinterval_s [Error]: RangeCheck
SLI [4] pstack
6
0
(linux)
(firefox)
```

The problem turns out to be the following line in `helpinit.sli`:
```
statusdict/hostos :: 6 Take (darwin) eq
```
If `statusdict/hostos ::` returns something shorter than 6 characters, we are in trouble. I will create a PR shortly.

**Comment by jougs** [comment]
Fixed by #518.
